docs/src/main/sphinx/connector/join-pushdown-enabled-true.fragment

```diff
@@ -0,0 +1,33 @@
+Cost-based join pushdown
+""""""""""""""""""""""""
+
+The connector supports cost-based :ref:`join-pushdown` to make intelligent
+decisions about whether to push down a join operation to the data source.
+
+When cost-based join pushdown is enabled, the connector only pushes down join
+operations if the available :doc:`/optimizer/statistics` suggest that doing so
+improves performance. Note that if no table statistics are available, join
+operation pushdown does not occur to avoid a potential decrease in query
+performance.
+
+The following table describes catalog configuration properties for
+join pushdown:
+
+.. list-table::
+  :widths: 30, 40, 30
+  :header-rows: 1
+
+  * - Property name
+    - Description
+    - Default value
+  * - ``join-pushdown.enabled``
+    - Enable :ref:`join pushdown <join-pushdown>`. Equivalent :ref:`catalog
+      session property <session-properties-definition>` is
+      ``join_pushdown_enabled``.
+    - ``true``
+  * - ``join-pushdown.strategy``
+    - Strategy used to evaluate whether join operations are pushed down. Set to
+      ``AUTOMATIC`` to enable cost-based join pushdown, or ``EAGER`` to
+      push down joins whenever possible. Note that ``EAGER`` can degrade
```

Comment:
Let's be more explicit. I've seen people confused with EAGER.

```suggestion
      push down joins without considering statistics if possible. Note that ``EAGER`` can degrade
```

My intent is to emphasise that "if possible" != "always". Not sure my suggestion does that - open to better ones.

Reply:
Reworded a bit, PTAL

Reply:
Very nice.